src/core/ext/transport/chttp2/server/chttp2_server.cc

```diff
@@ -966,6 +966,7 @@ grpc_error_handle Chttp2ServerAddPort(Server* server, const char* addr,
                                                     args_modifier);
   }
   *port_num = -1;
+  absl::StatusOr<std::vector<grpc_resolved_address>> resolved_or;
```

Comment:
Please see https://abseil.io/tips/181 for tips on avoiding the `_or` suffix.

Reply:
done.